ocaml/xapi/xapi_xenops.ml

```diff
@@ -28,6 +28,20 @@ module Rrdd = Rrd_client.Client
 open Xenops_interface
 open Xapi_xenops_queue
 
+let assert_final_power_state_is' ~__context ~self ~expected =
```

Comment:
Can you make this name a little clearer? It's not really an "assert" anymore. (Also you don't really need the `'`).